stuff/profiles/layouts/rooms/Default/menubar_template.xml

```diff
@@ -307,6 +307,7 @@
         <separator/>
         <command>MI_OpenFileBrowser</command>
         <command>MI_OpenFileBrowser2</command>
+	<command>MI_OpenAdvancedColorSelector</command>
```

Comment:
To match the location of if in code, please move this up below `MI_OpenColorModel`